ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/activity/PaywallContract.kt

```diff
@@ -0,0 +1,31 @@
+package com.revenuecat.purchases.ui.revenuecatui.activity
+
+import android.app.Activity
+import android.content.Context
+import android.content.Intent
+import android.os.Build
+import androidx.activity.result.contract.ActivityResultContract
+import com.revenuecat.purchases.PurchasesError
+import com.revenuecat.purchases.PurchasesErrorCode
+
+internal class PaywallContract : ActivityResultContract<PaywallActivityArgs, PaywallResult>() {
```

Comment:
I'd love a docstring here too

Reply:
Added in https://github.com/RevenueCat/purchases-android/pull/1317/commits/09c51a8fd2ddd35d952d1d2910f31cbd280b3f9b